tests/test_rewards.py

```diff
@@ -120,7 +120,9 @@ def test_positive_max_penalty_raises_value_error(self):
 
     def test_zero_max_penalty_returns_zero(self):
         reward_fn = get_repetition_penalty_reward(ngram_size=2, max_penalty=0.0)
-        self.assertEqual(reward_fn, 0)
+        completions = [[{"content": "test test test"}]]
+        rewards = reward_fn(completions)
+        self.assertEqual(rewards, [0.0])
```

Comment:
This test was failing, because it was comparing int to function. I guess this was the intention

Reply:
you can remove this test